src/procedure/nodevalue.h

```diff
@@ -61,4 +62,9 @@ class NodeValue
     double asDouble();
     // Return value represented as a string
     std::string asString(bool addQuotesIfRequired = false) const;
+
+    // Express as a tree node
```

Comment:
I'm a sucker for grouping functions together in headers under big banner comments to aid readability and casual location of functions, as I'm sure you've noticed! So...
```suggestion

    /*
     * Serialisable
     */
    public:
    // Express as a tree node
```